src/styles/_mixins.scss

```diff
@@ -34,7 +34,7 @@
   }
 }
 
-@mixin scrollbar($width: 5px, $trackbg: $color-page-background, $thumb-color: $color-accent) {
+@mixin scrollbar($width: 5px, $trackbg: var(--scrollbar-track-background), $thumb-color: var(--scrollbar-thumb-color)) {
   -webkit-overflow-scrolling: touch;
```

Comment:
what's the difference here between `$color-accent` and `var(--scrollbar-thumb-color)`

Reply:
So a lot of these variables are kind of passed around so that you can theme the application, the reason this is set this way is so that the end user can change the accent color, and the scollbar default thumb color would be the accent color, but if they wanted a different color for the scrollbar, they would change the value of `--scrollbar-thumb-color`.